modules/instance_template/versions.tf

```diff
@@ -17,6 +17,11 @@
 terraform {
   required_version = ">=1.3"
   required_providers {
+    google = {
+      source  = "hashicorp/google"
```

Comment:
Could we use the beta provider to create the new resources as this could be a breaking change if there are other modules in the root module that have a tpb < 5.36 constraint.

Reply:
Done!